### DISCUSSION TITLE
Choosing a license

### DISCUSSION THREAD

**Discussion by Buduf** [Buduf/PTI-Ear-Trainer#2] [General]
We must include a license in the projec, if we dont want to restrict everything.
On [choosealicense.com](https://choosealicense.com/) I suggest we can use the MIT license, which is the simplest and the most lenient.

**Comment by Buduf** [comment]
OK. I uploaded the license, but I was such an idiot that I didn't replaced the [year] and [full name] in the file. @HunBonus Since the idea was yours, can you update the file with your name?

**Reply by Falesz** [reply]
Sure, I'll take a look at it soon.
